### PR TITLE
Cleans docs a bit for Expenses section. Not comprehensive. #821

### PR DESCRIPTION
**Pull request by knod**

### CODE REVIEW

File: src/forms/CurrentExpenses.js [hunk 1]
```diff
@@ -243,19 +243,16 @@ const HousingRadio = function ({ currentValue, label, time, updateClientValue })
...
  * @function
  * @param {object} props
- * @param {object} props.current - Client data of current user circumstances
- * @param {string} props.type - 'expense' or 'income', etc., for classes
- * @param {string} props.time - 'current' or 'future'
- * @param {function} props.updateClientValue - Sets state values
+ * @param {object} props.current Client data of current user circumstances
+ * @param {string} props.type 'expense' or 'income', etc., for classes
+ * @param {string} props.time 'current' or 'future'
+ * @param {function} props.updateClientValue Sets state values
```
Comment (ethanbb) [inline src/forms/CurrentExpenses.js]: Hmm, I was using the dashes for params in `App.js`. Do you want to not use the dashes?

Comment (knod) [reply src/forms/CurrentExpenses.js]: I'm not sure how the dashes show up in the docs, but it seems to me like they might turn out looking weird. I think a syntax highlighter like the github one would help at least _me_ not to need the dashes to visually separate the description visually from the var name.

I'll go ahead and merge!

Comment (knod) [reply src/forms/CurrentExpenses.js]: But we can keep talking here if you still have thoughts.

Comment (knod) [reply src/forms/CurrentExpenses.js]: Nice find!

**Pull request merged**